### PR TITLE
Upgrade Java 8/11 to 8u222/11.0.4 (from 8u212/11.0.3)

### LINKED ISSUE #387: Security update for distroless java
Hello,

If I understand you follow Debian packaging. Debian have publish [security upgrade](https://www.debian.org/security/2019/dsa-4486) 3 days ago (more infos in [openjdk update](https://openjdk.java.net/groups/vulnerability/advisories/2019-07-16)).

If I see [GCR](https://console.cloud.google.com/gcr/images/distroless/GLOBAL/java?gcrImageListsize=15) it seems to distroless don't apply the update.

Do you have any plan for apply this update ? Or did you have documentation how can I create a PR for it ?

### PR DESCRIPTION
**Pull request by chanseokoh**
Fixes #387.

On top of #389, so #389 should be merged first.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ jdk:
 install:
   - export PATH=$PATH:$HOME/bin && mkdir -p $HOME/bin
   - wget https://github.com/bazelbuild/buildtools/releases/download/0.22.0/buildifier && chmod +x buildifier && mv buildifier $HOME/bin/
-  - wget https://github.com/bazelbuild/bazel/releases/download/0.24.0/bazel-0.24.0-linux-x86_64 && mv bazel-0.24.0-linux-x86_64 bazel && chmod +x bazel && mv bazel $HOME/bin/
+  - wget https://github.com/bazelbuild/bazel/releases/download/0.28.1/bazel-0.28.1-linux-x86_64 && mv bazel-0.28.1-linux-x86_64 bazel && chmod +x bazel && mv bazel $HOME/bin/
```
Comment (loosebazooka) [inline .travis.yml]: I feel like maybe this should be in a different PR?

File: package_manager/parse_metadata.py [hunk 2]
```diff
@@ -22,7 +22,7 @@
 def parse_package_metadata(data, mirror_url, snapshot, package_prefix):
     """ Takes a debian package list, changes the relative urls to absolute urls,
     and saves the resulting metadata as a json file """
-    raw_entries = [line.decode('utf-8').rstrip() for line in data.splitlines()]
+    raw_entries = [line.rstrip() for line in data.splitlines()]
```
Comment (loosebazooka) [inline package_manager/parse_metadata.py]: why this change?

File: .travis.yml (hunk 1 above)
Comment (chanseokoh) [reply .travis.yml]: Okay, I'll break this PR into two.

Comment (chanseokoh) [reply .travis.yml]: Created #389. @donmccasland @sharifelgamal please review #389 instead of this, which contains these changes unrelated to Java upgrade.

File: package_manager/parse_metadata.py (hunk 2 above)
Comment (chanseokoh) [reply package_manager/parse_metadata.py]: Explained in #389.
